src/test/java/com/wdtinc/mapbox_vector_tile/build/MvtBuildTest.java

```diff
@@ -74,7 +75,7 @@ public void testPoints() throws IOException {
 
         assertNotNull(bytes);
 
-        JtsMvt expected = new JtsMvt(singletonList(new JtsLayer(TEST_LAYER_NAME, tileGeom.mvtGeoms)));
+        JtsMvt expected = new JtsMvt(singletonList(new JtsLayer(TEST_LAYER_NAME, tileGeom.mvtGeoms, Extent.DEFAULT)));
```

Comment:
Don't need to modify all the tests with a 3-arg if you remove the breaking API change to the constructors to the layer.